libraries/standard/http/src/http_client.c

```diff
@@ -1465,6 +1470,13 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
                     rangeStartOrlastNbytes, rangeEnd ) );
         returnStatus = HTTP_INVALID_PARAMETER;
     }
+    else if( rangeStartOrlastNbytes == INT32_MIN )
```

Comment:
I didn't see a unit test update here? Is this needed?

Reply:
Oh this was for AddRangeHeader and the unit was updated in the other PR. The reason this is here is because I created a branch from that branch. Anyway, I have rebased and this is no longer in this PR.